frontend/main.js

```diff
@@ -122,6 +124,7 @@ async function startApp () {
     },
     computed: {
       showNav () {
+        console.log(':: groupsByName', this.$store.getters.groupsByName)
```

Comment:
Should this be here? Either should be removed before being merged, or reasoning should be given for why this is being logged. Also, this is debug logging, so `console.debug` should be used

Reply:
was only for debugging while working on this. removed